### DISCUSSION TITLE
[REQUEST] Markdown link must be produce

### DISCUSSION THREAD

**Discussion by pprados** [Textualize/rich#3449] [Q&A]
Consider posting in https://github.com/textualize/rich/discussions for feedback before raising a feature request.

Have you checked the issues for a similar suggestions?

**How would you improve Rich?**

```
import rich.console
import rich.markdown
rich.console.Console(emoji=True).print(rich.markdown.Markdown("[a](www.toto.fr)"))
```
Produce
```
a
```

Add an option, or the defaut to produce
```
[a](www.toto.fr)
```
with link format

**What problem does it solve for you?**

All markdown links are hidden.

The code has all the information it needs via the parser

**Comment by TomJGooding** [comment]
> All markdown links are hidden.

Just to clarify, this will be rendered as a hyperlink if your terminal supports this feature.

You can disable this behaviour with `Markdown(hyperlinks=False)`.

**Comment by willmcgugan** [comment]
@pprados Not all terminals support inline links. Notably macOS terminal.app. If you are on macOS, we recommend iTerm
